commands/botAdmin.js

```diff
@@ -147,17 +147,61 @@ const Module = new Augur.Module()
         .addField("Shard Uptime", `${Math.floor(client.uptime / (24 * 60 * 60 * 1000))} days, ${Math.floor(client.uptime / (60 * 60 * 1000)) % 24} hours, ${Math.floor(client.uptime / (60 * 1000)) % 60} minutes`, true)
         .addField("Shard Commands Used", `${client.commands.commandCount} (${(client.commands.commandCount / (client.uptime / (60 * 1000))).toFixed(2)}/min)`, true)
         .addField("Total Memory", `${mem}MB`, true);
-
-        msg.channel.send({embed:embed});
+        fetch("https://srhpyqt94yxb.statuspage.io/api/v2/summary.json").then(res => res.json())
+            .then((json) => {
```

Comment:
`fetch` isn't defined anywhere. For consistency sake, recommend requiring `axios` at the top and using that.

In addition, using `.then()` in an `async` function is generally seen as bad form. Better to use `await` here.

Reply:
oh, I was using node fetch. Sorry about that. I don't know axios.